pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationMultiValueQueriesTest.java

```diff
@@ -157,7 +157,7 @@ public void testCastCountMV() {
     Assert.assertEquals(brokerResponse.getResultTable().getDataSchema().getColumnName(0), "cnt_column6");
 
     brokerResponse = getBrokerResponseForSqlQueryWithFilter(query);
-    QueriesTestUtils.testInterSegmentResultTable(brokerResponse, 62480L, 874176L, 62480L, 400000L,
```

Comment:
Why this value get changed? Did we change the underlying data or query?

Reply:
This value changes because with the AND/OR flattening, the filter is compiled the same way in SQL and PQL, thus this value becomes the same as compiling with PQL. This value increases by chance, not always.